d2/src/main/java/com/linkedin/d2/xds/XdsChannelFactory.java

```diff
@@ -53,6 +54,7 @@ public ManagedChannel createChannel()
     }
 
     return builder.keepAliveTime(5, TimeUnit.MINUTES)
+        .proxyDetector(GrpcUtil.NOOP_PROXY_DETECTOR)
```

Comment:
maybe add a comment issues without this?

Reply:
Done.